### DISCUSSION TITLE
Undocking during the Minds Unraveled can lead to unlimited debilitating hallucinations

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14921] [Bug reports]
### What happened?

The hallucinating_mindsunraveled affliction gives really bad visual effects until the minds unraveled event is concluded. If the sub undocks before the event finishes, then anyone with the affliction now has this debilitating affliction indefinitely unless cheated away.

This is not good, unsurprisingly

![image](https://github.com/user-attachments/assets/915b64f5-f000-4948-a997-46fcb28ed7ff)

While the idea of going permanently insane is quite fitting for the world of barotrauma, I don't think players will like it

### Reproduction steps

1. Start the minds unraveled event in multiplayer
2. advance until you have the special hallucinating affliction
3. get a crew member to undock
4. observe that you now have indefinite hallucinations

**Comment by JeroonBosch** [comment]
> While the idea of going permanently insane is quite fitting for the world of barotrauma, I don't think players will like it

players just don't know what's good gameplay

joking aside, thanks for the report, a fix is in the works

**Comment by Regalis11** [comment]
This has been fixed in the latest hotfix, closing.
